eventhandling/getSliEvent.go

```diff
@@ -148,13 +161,6 @@ func retrieveMetrics(eventData *keptnv2.GetSLITriggeredEventData, keptnHandler *
 				Success: false,
 				Message: err.Error(),
 			})
-		} else if math.IsNaN(sliValue) {
-			sliResults = append(sliResults, &keptnv2.SLIResult{
-				Metric:  indicator,
-				Value:   0,
-				Success: false,
-				Message: "SLI value is NaN",
-			})
```

Comment:
Do we no longer need this case?

Reply:
Never mind, I see this is now handled in `GetSLIValue` with a `|| math.IsNaN(floatValue)`